client/transactions/list/test/__snapshots__/index.tsx.snap

```diff
@@ -131,15 +131,15 @@ exports[`Transactions list renders correctly when can filter by several currenci
                         aria-haspopup="true"
                         autocomplete="off"
                         class="woocommerce-select-control__control-input"
-                        id="woocommerce-select-control-15__control-input"
+                        id="woocommerce-select-control-16__control-input"
                         placeholder=""
                         role="combobox"
                         type="text"
                         value=""
                       />
                       <span
                         class="screen-reader-text"
-                        id="search-inline-input-15"
+                        id="search-inline-input-16"
```

Comment:
These changes to the rendered HTML `id` numbers can be ignored – this occurs because I've added a new test case being rendered, bumping many of these ids by +1.